packages/odata-v2/src/request-builder/get-all-request-builder.spec.ts

```diff
@@ -47,6 +48,41 @@ describe('GetAllRequestBuilder', () => {
   });
 
   describe('url', () => {
+    it('should set ascending order', async () => {
+      const expected =
+        '/testination/sap/opu/odata/sap/API_TEST_SRV/A_TestEntity?$orderby=ComplexTypeProperty/StringProperty%20asc';
+      const request = await testEntityApi
+        .requestBuilder()
+        .getAll()
+        .orderBy(asc(testEntityApi.schema.COMPLEX_TYPE_PROPERTY.stringProperty))
+        .url(defaultDestination);
+      expect(request).toBe(expected);
+    });
+
+    it('should set descending order', async () => {
+      const expected =
+        '/testination/sap/opu/odata/sap/API_TEST_SRV/A_TestEntity?$orderby=ComplexTypeProperty/StringProperty%20desc';
+      const request = await testEntityApi
+        .requestBuilder()
+        .getAll()
+        .orderBy(
+          desc(testEntityApi.schema.COMPLEX_TYPE_PROPERTY.stringProperty)
+        )
+        .url(defaultDestination);
+      expect(request).toBe(expected);
+    });
+
+    it('should set ascending order when any order not specified', async () => {
```

Comment:
[req] Could you add one more test where two properties are given where one has nothing (default should be asc) and the other one has explicitly `desc` set.

orderBy(
          desc(prop1,desc(prop2))
        )

to see that the looping over the elemts wokrs if the types are mixed.

Reply:
Is the additonal test will be `orderBy(prop1, desc(prop2))`?
I think the `prop1,desc(prop2)` cannot be wrapped by `desc` because `asc` and `desc` takes a single argument.

Reply:
oh, yes typo from my side. Only one desc

Reply:
I added the test in V2 package